### PR TITLE
fix Otto example and some typos

### PR DESCRIPTION
**Pull request by tonysm**
## Fixed

- otto first example

## Added

- example using a Redis dependency locally

### CODE REVIEW

File: _posts/2015-11-19-otto-first-impressions.md [hunk 1]
```diff
@@ -231,9 +226,78 @@ customization "docker" {
...
+> The Consul foundation installs a local Consul server in your development environment and configures it to respond to \*.consul DNS queries. [read more](https://ottoproject.io/docs/foundations/consul.html)
+
+Let's use it. First, edit the welcome view:
+
+```html
+@extends('_layout')
+
+@section('content')
+    <div class="content">
+        <div class="title">Counts: {{ '{{ $' }}counts }}</div>
```
Comment (hannesvdvreken) [inline _posts/2015-11-19-otto-first-impressions.md]: This correct?

**Comment by hannesvdvreken** [comment]
:+1:

File: _posts/2015-11-19-otto-first-impressions.md (hunk 1 above)
Comment (tonysm) [inline _posts/2015-11-19-otto-first-impressions.md]: yes, it's the jekyll escaping..

**Comment by tonysm** [comment]
will merge it :)

**Pull request merged**